quickwit/quickwit-serve/src/rest.rs

```diff
@@ -44,6 +46,10 @@ use crate::search_api::{search_get_handler, search_post_handler, search_stream_h
 use crate::ui_handler::ui_handler;
 use crate::{with_arg, BodyFormat, QuickwitServices};
 
+/// The minimum size a response body must be in order to
+/// be automatically compressed with gzip.
+const MINIMUM_RESPONSE_COMPRESSION_SIZE: u16 = 10 << 10;
```

Comment:
Sounds reasonable. Do you have a rule of thumb to estimate the  right `MINIMUM_RESPONSE_COMPRESSION_SIZE`?

Reply:
Not really, it's a fairly arbitrary size, but a size derived so that the general REST API requests won't get needlessly compressed, and small searches won't either while catching the bigger things.